Formula/kn.rb

```diff
@@ -0,0 +1,37 @@
+class Kn < Formula
+  desc "Command-line interface for managing Knative Serving and Eventing resources"
+  homepage "https://github.com/knative/client"
+  url "https://github.com/knative/client.git",
+      tag:      "v0.24.0",
+      revision: "0bbb3ec9d92665d372d8393ef3036ba7e0c069a4"
+  license "Apache-2.0"
+
+  depends_on "bash" => :build
+  depends_on "go" => :build
```

Comment:
Is there a way to specify the go version, too ? Asking because there has been some CVEs recently that are connected to certain go versions (including patch level).

Reply:
yes, the version can be pinned like so `depends_on "go@1.16" => :build`, but `brew audit` does not like it and wants us to use just `go`. 

```
* Dependency 'go@1.16' is an alias; use the canonical name 'go'.
```

Reply:
I see, so I guess that a `brew upgrade` will update `kn` also when only the base go version has been updated (that is what I would expect).

Reply:
yes. it would use the latest go version they support to build kn.

Reply:
my concern was just that a `brew upgrade` will consider a rebuild if only the go version changed but not the formula itself.

Reply:
oh no.. these build instructions are executed by the homebrew ci and homebrew delivers only binaries to users, unless the user does `brew install --build-from-source kn` to build the package locally.

So `brew upgrade` will not do a rebuild if the go version changed.